src/site/content/en/fast/codelab-prefetching-wiih-quicklink/index.md

```diff
@@ -0,0 +1,190 @@
+
+import Footer from '@components/Footer';
+import Hero from '@components/Hero';
+import style from './index.module.css';
+import { withQuicklink } from 'quicklink/dist/react/hoc.js';
+
+const Home = lazy(() => import(/* webpackChunkName: "home" */ '@pages/Home'));
+const About = lazy(() => import(/* webpackChunkName: "about" */ '@pages/About'));
+const Article = lazy(() => import(/* webpackChunkName: "article" */ '@pages/Article'));
+const Blog = lazy(() => import(/* webpackChunkName: "blog" */ '@pages/Blog'));
+```
+
+Next, create an `options` object to use as an argument when calling `quicklink`:
+
+```javascript/0-2/
+const options = {
+	origins: []
+};
+```
+Finally, wrap each route with the `withQuicklink()` higher order component, passing it an `options` parameter and the target component for that route:
+
+```javascript/5-11/
+const App = () => (
+  <div className={style.app}>
+    <Hero />
+    <main className={style.wrapper}>
+      <Suspense fallback={<div>Loading...</div>}>
+        <Route path="/" exact component={withQuicklink(Home, options)} />
+        <Route path="/blog" exact component={withQuicklink(Blog, options)} />
+        <Route
+          path="/blog/:title"
+          component={withQuicklink(Article, options)}
+        />
+        <Route path="/about" exact component={withQuicklink(About, options)} />
+      </Suspense>
+    </main>
+    <Footer />
+  </div>
+);
+```
+The previous code instructs to prefetch chunks for the routes wrapped with `withQuicklink()`, when the link comes into the view.
+
+{% Aside %}
+If the Glitch console throws an error at this point about the lack of a dependency, try running `refresh` to restart the server and run the `npm run build` after that.
+{% endAside %}
+
+## Measure again
+
+1. Open the site in a new Chrome tab.
+{% Instruction 'devtools-network', 'ol' %}
+{% Instruction 'disable-cache', 'ol' %}
+1. In the Throttling drop-down list, select **Fast 3G** to simulate a slow connection type.
+{% Instruction 'reload-app', 'ol' %}
+1. Filter requests for "chunk" in the network panel.
+
+When the home page loads the chunks for that route are loaded. After that, `quicklink` prefetches the route's chunks for the in-viewport links:
+
+<img class="w-screenshot" src="./cdt-home-optimized.png" alt="Network panel showing the home page prefetching chunks.">
+
+These chunks are requested at the lowest priority and without blocking the page.
+
+Next: 
+
+1. Clear the network panel.
+1. Click on the **Blog** link to navigate to that page.
+1. Filter requests for "chunk" in the network panel.
+
+<img class="w-screenshot" src="./cdt-blog-optimized.png" alt="Network panel showing the blog page with chunks picked up from cache.">
+
+The **Size** column indicates that these chunks were retrieved from the "prefetch cache", instead of the network. Loading these chunks without a `quicklink` took approximately **580ms**. Using the library it now takes **2ms**, which represents a **99% reduction**!
```

Comment:
```suggestion
The **Size** column indicates that these chunks were retrieved from the "prefetch cache", instead of the network. Loading these chunks without a Quicklink took approximately **580ms**. Using the library it now takes **2ms**, which represents a **99% reduction**!
```

Reply:
I think this isn't working because we still have the **Disable cache** checkbox enabled. I think you need to update the instructions to account for that.